src/graph/backend/dnnl/kernels/sdp.hpp

```diff
@@ -65,17 +68,25 @@ struct sdp_base_t : public kernel_base_t {
         if (enable_ukernel) {
             kernel = std::make_shared<sdp_primitive_kernel_t<quantized>>();
             ret = kernel->compile_impl(part, g_engine, inputs, outputs);
+            if (ret == status::success)
+                VDISPATCH_GRAPH_SDP("dispatch to sdp_primitive_kernel_t");
         }
 
         if (ret != status::success && enable_decomp) {
             kernel = std::make_shared<sdp_decomp_kernel_t<quantized, dt>>();
             ret = kernel->compile_impl(part, g_engine, inputs, outputs);
+            if (ret == status::success)
+                VDISPATCH_GRAPH_SDP("dispatch to sdp_decomp_kernel_t");
         }
 
         if (ret != status::success) {
             kernel = std::make_shared<larger_partition_kernel_t>();
             ret = kernel->compile_impl(part, g_engine, inputs, outputs);
+            if (ret == status::success)
+                VDISPATCH_GRAPH_SDP("dispatch to larger_partition_kernel_t");
         }
+        if (ret != status::success)
+            VDISPATCH_GRAPH_SDP("fail to dispatch to sdp kernel");
```

Comment:
I'm thinking to simplify the checks. Is it possible to do the check only before return?
```
if (ret == status::success)
    VDISPATCH_GRAPH_SDP("sdpa is dispatched to (%s)", kernel->str());
else
    VDISPATCH_GRAPH_SDP("sdpa is failed to dispatch");
```